src/Entity/MatchingContext.php

```diff
@@ -115,12 +115,19 @@ class MatchingContext
     private $description;
 
     /**
-     * @var string
+     * @var string a CSS selector
```

Comment:
Shouldn’t we just remove this unused field ?

Reply:
Actually, we use private note to tell more about what the matching strategy etc.
So I agree, we can delete this filed description in matching contexte area @JalilArfaoui

Reply:
I was thinking of the (never used nor shown) `css selector` field @MaartenLMEM

Reply:
ah ! :shipit:

Reply:
I'd like to keep it and extends the matching strategy to an hybrid one in the future. CSS selector are less invasive and could be used in conjunction with the `declarativeContent` API.
I just added a descriptions since I find the naming confusing.

Reply:
I agree about thinking about an eventual `declarativeContent` use 

But I don’t see how XPath is less *invasive* … ?

Reply:
> But I don’t see how XPath is less invasive … ?
XPath __is__ more invasive.